docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -362,17 +386,17 @@ _build/validator/rel/miner/bin/miner start
 One last step. When building from source, you'll need to manually load the genesis block once your miner is deployed and running. To do this, run the following:
 
 ```
-wget https://snapshots.helium.wtf/genesis.testnet
+wget https://snapshots.helium.wtf/genesis.mainnet
```

Comment:
I don't think the `*.helium.wtf` FQDN is right for mainnet. But I don't know where the real snapshots are going to live. Ask Evan?

Reply:
Have to leave these as plugs until the URLs are clarified.